tensornetwork/tn_keras/mpo.py

```diff
@@ -0,0 +1,203 @@
+import tensorflow as tf
+from tensorflow.keras.layers import Layer  # type: ignore
+from tensorflow.keras import activations
+from tensorflow.keras import initializers
+from typing import List, Optional, Text, Tuple
+import tensornetwork as tn
+from tensornetwork.network_components import Node
+import numpy as np
+import math
+
+
+class DenseMPO(Layer):
```

Comment:
You can register this with the Keras object serialization logic using register_keras_serializable(). Then you don't need to add it to the custom object dicts below.

Reply:
Nice, this makes model loading much cleaner - updated.